homeworks/06-advanced-redux/src/components/PizzaBasketItem.tsx

```diff
@@ -27,12 +28,12 @@ export function PizzaBasketItem({ onMinus, _id, price, name, count }: PizzaProps
                          alt="pizzapic" />
                 </div>
                 <div className="w-full p-4">
-                    <PizzaName name={name} />
+                    <ShowPizzaName name={name} />
                     <PizzaDescription desc="description" />
-                    <PizzaCount count={count} />
+                    <ShowPizzaCount count={count} />
```

Comment:
The value should be rounded:
<img width="532" alt="Screenshot 2021-05-20 at 15 40 08" src="https://user-images.githubusercontent.com/364880/118986140-c0b8eb80-b987-11eb-81d1-eaaed2f31002.png">